TVUIComponent/lib/src/main/java/com/amazon/android/tv/tenfoot/presenter/CardPresenter.java

```diff
@@ -187,8 +198,8 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
                         public void onResourceReady(Bitmap resource, GlideAnimation<? super Bitmap> glideAnimation) {
                             cardView.setInfoAreaBackground(infoFieldWithProgressBarBackground);
                             Bitmap bitmap = Helpers.addProgressToThumbnail((Activity) mContext, resource, playbackPercentage, 0);
-                            cardView.getMainImageView().setImageBitmap(bitmap);
-                        }
+                            ImageView imageView = cardView.findViewById(R.id.main_image);
```

Comment:
Why do we need to call 'findViewById' here and in few places below, instead of using 'getMainImageView()', that returns  a cached reference to the image view?